core/schema_utils.py

```diff
@@ -67,9 +68,22 @@
 SCHEMA_TYPE_BASESTRING = 'basestring'
 SCHEMA_TYPE_UNICODE_OR_NONE = 'unicode_or_none'
 SCHEMA_TYPE_OBJECT_DICT = 'object_dict'
+SCHEMA_TYPE_WEAK_MULTIPLE = 'weak_multiple'
 
 SCHEMA_OBJ_TYPE_SUBTITLED_HTML = 'SubtitledHtml'
 SCHEMA_OBJ_TYPE_SUBTITLED_UNICODE = 'SubtitledUnicode'
+ALL_SCHEMAS: Dict[str, Tuple[type, ...]] = {
+    SCHEMA_TYPE_BOOL: (bool,),
+    SCHEMA_TYPE_DICT: (dict,),
+    SCHEMA_TYPE_DICT_WITH_VARIABLE_NO_OF_KEYS: (dict,),
+    SCHEMA_TYPE_FLOAT: (float,),
+    SCHEMA_TYPE_HTML: (str, bytes),
```

Comment:
Should we allow `bytes` here?

Reply:
Umm, I just added it for the sake of completeness. Should I get rid of it?

Reply:
I think if it works without it then yes

Reply:
Okay